### ISSUE TITLE
Cannot read property '0' of undefined

### ISSUE THREAD

**Issue by dimylik** [fraywing/textAngular#1376]
"textangular": "^1.5.12",

this function:
scope.handlePopoverEvents = function() {
	if (scope.displayElements.popover.css('display')==='block') {
		if(_resizeTimeout) $timeout.cancel(_resizeTimeout);
			_resizeTimeout = $timeout(function() {
			//console.log('resize', scope.displayElements.popover.css('display'));
			scope.reflowPopover(_resizeElement);
			scope.reflowResizeOverlay(_resizeElement);
		}, 100);
	}
};

calling scope.reflowPopover(_resizeElement);
where _resizeElement is undefined
and later "_el[0].offsetTop-scrollTop.top" returning "Cannot read property '0' of undefined"

HTML:
<text-angular id="addSummary" required name="addSummary" ng-model="addReport.report.summary"
              ta-toolbar="[['bold','italics'],['justifyLeft','justifyCenter','justifyRight','justifyFull'],['ol','ul']]"
              ta-toolbar-button-class="btn xx-text-editor-btn"
              ta-toolbar-group-class="xx-text-editor-btn-group"></text-angular>

CSS:
.xx-text-editor-btn{
	background:none;
	border:none;
	color: lighten(@xx-dark-col-h,2%);
	&:hover{
		color: @xx-dark-col-h;
	}
}
.xx-text-editor-btn-group{
	float:left;
	border-right:solid 1px @xx-text-editor-btn-group-border-color;
	padding:0 10px;
	margin-top:5px;
}

**Comment by stevokk** [comment]
If you would like someone to investigate, I'd suggest creating a demo to show what issue you're encountering. Nobody is going to manually recreate your project from the above details.

**Comment by alyrik** [comment]
+1. The same for me. There is no error in v1.5.11. May be the problem is raising only when using custom popovers.

**Comment by ksokol** [comment]
Today I run into the same issue while updating textAngular from version 1.3.6 to 1.5.16. The good news: it is reproducible with [textAngular.com.html](https://github.com/fraywing/textAngular/blob/ff8e48087f780d30f54e77b06f09e0b85f9517e9/demo/textAngular.com.html).

This bug only occurs by excluding textAngular.css **AND** including jQuery.

### Explanation

Initially, the popover  is hidden by [CSS](https://github.com/fraywing/textAngular/blob/ff8e48087f780d30f54e77b06f09e0b85f9517e9/src/textAngular.css#L114). Even if `textAngular.css` is left out, jqLite evaluates [this expression](https://github.com/fraywing/textAngular/blob/ff8e48087f780d30f54e77b06f09e0b85f9517e9/src/main.js#L124) to false. But as soon as you include jQuery the same expression evaluates to true. 

Because of [scope.resizeElement](https://github.com/fraywing/textAngular/blob/ff8e48087f780d30f54e77b06f09e0b85f9517e9/src/main.js#L128 ) is not defined at this point in time, a `Cannot read property '0' of undefined` is thrown while accessing the first element in [_el](https://github.com/fraywing/textAngular/blob/ff8e48087f780d30f54e77b06f09e0b85f9517e9/src/main.js#L223).

### Workaround

Add 
```
text-angular .popover {
   display: none;
}
```

to your CSS. At least in my application no error occured after that.

Steps to reproduce with `textAngular.com.html`

1. Remove `<link rel="stylesheet" href="http://textangular.com/dist/textAngular.css" type="text/css">`
2. Include jQuery right before `<script src="http://ajax.googleapis.com/ajax/libs/angularjs/1.3.11/angular.min.js" type="text/javascript"></script>`
3. Remove `<link rel="stylesheet" href="http://netdna.bootstrapcdn.com/bootstrap/3.1.0/css/bootstrap.min.css">` because it contains `.popover { display: none; }` too

Open the HTML and scroll up and down. In the console you will see the error.

**Comment by alyrik** [comment]
Yes, workaround seems to help. Thank you!

**Comment by ckelley7** [comment]
There are two ways to instantiate textAngular:

`<text-angular ng-model="htmlVariable"></text-angular>`

OR

`<div text-angular ng-model="htmlVariable"></div>`

If you are like me and use the latter, you'll need a slightly different CSS selector to for the workaround specified above:

`[text-angular] .popover {
    display: none;
  }`
